gitian-building/gitian-building-mac-os-sdk.md

```diff
@@ -53,4 +53,5 @@ mv MacOSX10.11.sdk.tar.gz gitian-builder/inputs
 
 Troubleshooting
 ---------------
-See [README_osx.md](https://github.com/bitcoin/bitcoin/blob/master/doc/README_osx.md) for troubleshooting tips.
+See [build-osx.md](See [build-osx.md](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md#deterministic-macos-dmg-notes) for troubleshooting tips.
```

Comment:
This is not correct markdown syntax, it looks like a copy/paste error, both the opening and closing part of the link seem to be duplicated.